app/src/main/java/com/example/streammoviesapplication/utils/Constants.kt

```diff
@@ -0,0 +1,5 @@
+package com.example.streammoviesapplication.utils
+
+object Constants {
+    const val API_KEY = "5a3975e2ec5d680f7e12dc27018913ff"
```

Comment:
API_KEY should not be in git repo. You should extract it to local.properties. Research on how to do that